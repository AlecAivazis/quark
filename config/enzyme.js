// external imports
import Enzyme from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import browserEnv from 'browser-env'

// add a request animation frame polyfill
import raf from 'raf'
raf.polyfill()

// enable the react 16 adapter for enzyme
Enzyme.configure({ adapter: new Adapter() })

// add various browser mocks
browserEnv()
