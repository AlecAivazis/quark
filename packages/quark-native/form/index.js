import _TextInput from './TextInput';
export { _TextInput as TextInput };