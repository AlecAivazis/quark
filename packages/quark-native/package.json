{
    "name": "quark-native",
    "version": "0.1.4",
    "module": "index.js",
    "main": "index.js",
    "dependencies": {
        "quark-core": "^0.1.4"
    }
}
