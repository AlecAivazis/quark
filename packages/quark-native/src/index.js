// @flow
export * from './components'
export * from 'quark-core'
