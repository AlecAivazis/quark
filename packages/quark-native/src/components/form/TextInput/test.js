// external imports
import React from 'react'
import { Text } from 'react-native'
import { mount } from 'enzyme'
// local imports
import { TextInput } from 'quark-native'

it('renders', () => expect(mount(<TextInput />)).toMatchSnapshot())
