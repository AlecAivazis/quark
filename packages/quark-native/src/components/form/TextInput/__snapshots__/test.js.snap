// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<Input>
  <GetTheme>
    <TextInput
      autoCapitalize="sentences"
      autoComplete="on"
      autoCorrect={true}
      editable={true}
      keyboardType="default"
      multiline={false}
      numberOfLines={2}
      secureTextEntry={false}
      style={
        Object {
          "backgroundColor": "#ffffff",
          "borderColor": "#EAEAEE",
          "borderRadius": 2,
          "borderWidth": 1,
          "fontSize": 14,
          "height": 36,
          "paddingLeft": 12,
        }
      }
    >
      <input
        autoCapitalize="sentences"
        autoComplete="on"
        autoCorrect="on"
        className="rn-MozAppearance-97bpaa rn-appearance-30o5oe rn-borderTopLeftRadius-1iymjk7 rn-borderTopRightRadius-s2skl2 rn-borderBottomRightRadius-l5bh9y rn-borderBottomLeftRadius-101sy47 rn-borderTopStyle-1efd50x rn-borderRightStyle-14skgim rn-borderBottomStyle-rull8r rn-borderLeftStyle-mm0ijv rn-borderTopWidth-5kkj8d rn-borderRightWidth-13l2t4g rn-borderBottomWidth-qklmqi rn-borderLeftWidth-1ljd8xs rn-boxSizing-deolkf rn-fontFamily-14xgk7a rn-fontSize-1b43r93 rn-height-1r8g8re rn-paddingTop-wk8lta rn-paddingRight-9aemit rn-paddingBottom-1mdbw0j rn-paddingLeft-f727ji rn-resize-1dz5y72"
        dir="auto"
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyPress={[Function]}
        onSelect={[Function]}
        readOnly={false}
        spellCheck={true}
        style={
          Object {
            "backgroundColor": "rgba(255,255,255,1)",
            "borderBottomColor": "rgba(234,234,238,1)",
            "borderLeftColor": "rgba(234,234,238,1)",
            "borderRightColor": "rgba(234,234,238,1)",
            "borderTopColor": "rgba(234,234,238,1)",
          }
        }
        type="text"
      />
    </TextInput>
  </GetTheme>
</Input>
`;
